src/pages/Admin/Charity/Account/Balances.tsx

```diff
@@ -0,0 +1,68 @@
+import { AccountType } from "types/contracts";
+import { useAdminResources } from "pages/Admin/Guard";
+import { useAssetQuery } from "services/juno/account";
+import ContentLoader from "components/ContentLoader";
+import QueryLoader from "components/QueryLoader";
+import { humanize } from "helpers";
+
+export default function Balances({ type }: { type: AccountType }) {
+  const { id } = useAdminResources();
+  const queryState = useAssetQuery({ id, type });
+
+  return (
+    <QueryLoader
+      queryState={queryState}
+      messages={{ loading: <Skeleton />, error: "Failed to get balances" }}
+    >
+      {({ total, free, invested, symbol }) => (
+        <div className="grid grid-cols-3 gap-6">
+          <Amount title="Total value" symbol={symbol}>
+            {total}
+          </Amount>
+          <Amount title="Invested balances" symbol={symbol}>
+            {invested}
+          </Amount>
+          <Amount title="Free balances" symbol={symbol}>
+            {free}
+          </Amount>
+        </div>
+      )}
+    </QueryLoader>
+  );
+}
+
+type Props = {
+  classes?: string;
+  title: string;
+  symbol: string;
+  children: number;
+};
+
+function Amount({ classes = "", ...props }: Props) {
+  return (
+    <div
+      className={
+        classes +
+        " border border-prim py-4 px-5 rounded bg-orange-l6 dark:bg-blue-d6"
+      }
+    >
+      <p className="text-xs text-gray-d1 dark:text-gray mb-2 uppercase">
+        {props.title}
+      </p>
+      <span className="font-bold text-xl font-heading">
+        {humanize(props.children, 2)}{" "}
```

Comment:
@ap-justin using `props.children` like this is a bit confusing, would've been better to use the suggested `value: number` (or whatever other name) field, not to mention there would be no need for the closing `</Amount>` tag.

Reply:
I dont find this confusing jsx children is `tyepof ReactNode` which `number` is a member of

Reply:
Would still be shorter and clearer with the prop.